src/ws-inline-edit/ws-inline-edit.js

```diff
@@ -0,0 +1,115 @@
+import {React, Component} from '../imports';
+
+/**
+ * This class describes a Preact/React component which renders a inline-edit element.
+ * The inline-edit component can be used everywhere in block where you want to have ability to change value.
+ * As an example you can use it in div blocks, rows, tables.
+ */
+export class WSInlineEdit extends Component {
+  
+  /**
+   * @propTypes
+   * Types of properties
+   */
+  static propTypes = {
+    text: React.PropTypes.string,
+    onUpdate: React.PropTypes.func
+  };
+  
+  /**
+   * @defaultProps
+   * Create default onUpdate function to prevent errors if user don't use it
+   */
+  static defaultProps = {
```

Comment:
I could not find any reference to defaultProps in the docs of Preact. Did you test, if this is working?